app/views/previousYearUnderpayment.scala.html

```diff
@@ -44,18 +47,25 @@
 <div class="grid-row">
     <div class="grid grid-2-3">
         <div class="inner-block">
-            <p class="lede">@Messages("tai.previous.year.underpayment.para1")</p>
-            <p>@Messages(
-                    "tai.previous.year.underpayment.para2",
-                    viewModel.previousTaxYear.year.toString,
-                    viewModel.previousTaxYear.next.year.toString,
-                    viewModel.shouldHavePaid,
-                    viewModel.actuallyPaid
-                )</p>
-            <p>@Messages("tai.previous.year.underpayment.para3", viewModel.allowanceReducedBy, viewModel.amountDue)</p>
-            <p>@Messages("tai.previous.year.underpayment.para4", viewModel.allowanceReducedBy)</p>
-            <p>@Messages("tai.previous.year.underpayment.para5")</p>
+            <p>@messages("tai.previous.year.underpayment.p1", TaxYearRangeUtil.futureTaxYearRangeHtmlNonBreak(-1))</p>
 
+            <p>@messages("tai.previous.year.underpayment.you.owe")
+                </br>
```

Comment:
`<br>` s are not the best way to get space between content. Look at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br